Makefile

```
@@ -136,6 +137,7 @@ ifeq ($(KERNEL_NAME), $(filter $(KERNEL_NAME),DragonFly FreeBSD))
 BINARY_REL ?= bin
 INCLUDE_REL ?= include
 LIBRARY_REL ?= lib
+MAN_REL ?= man/man1
```

That should be share/man/man1.
```suggestion
MAN_REL ?= share/man/man1
```

---

Sorry for the late reply, forgotten here :)
Pushed the changes, I'm think the share/man/man1 should be used in the other places also, also pushed the change to add Haiku specific paths